cmd/commands/root.go

```diff
@@ -106,7 +107,11 @@ func PreValidationCheck(ctx context.Context, k8sclientset *k8sutil.Clientsets, o
 	}
 
 	rookVersion := exec.RunCommandInOperatorPod(ctx, k8sclientset, "rook", []string{"version"}, operatorNamespace, cephClusterNamespace, true, false)
+	re := regexp.MustCompile("(?m)[\r\n]+^.*go: go.*$") // remove the go version from the output
+	rookVersion = re.ReplaceAllString(rookVersion, "")
```

Comment:
How about factoring out this string replacement to a small helper method, then add a unit test for it? Whenever there is a regular expression, a unit test is always helpful.

Reply:
added. 

I want to add unit tests for all the methods but seems like that will take more time to release and require my bandwidth. XD

Reply:
Unit tests for now can avoid the high level commands since they are covered by the integration tests, but for small things like this, unit tests will be perfect.

Reply:
make sense